File: LinearSearch.c
#include<stdio.h>
int main()
{
    int size;
    printf("Enter the size of the array:\n");
    scanf("%d",&size);
    int arr[size];
    printf("Enter the array Elements:");
    for(int i=0;i<size;i++)
    {
        scanf("%d",&arr[i]);
    }
    printf("Enter the element to search:\n");
    int element;
    scanf("%d",&element);
    for(int i=0;i<size;i++)
    {
        if(arr[i]==element)
        {
            printf("%d is present at index %d",element,i);
            break;
        }
        if(i==size-1)
            printf("Element not found");
    }
}

File: linearSearch.py
def Search(n,arr,target):
    for i in range(n):
        if (arr[i]==target):
            return i
    return -1

if __name__ == "__main__":
    n = int(input("Enter the array size:"))
    arr = list(range(n))
    target = int(input("Enter the target element:"))
    for i in range(n):
        arr[i] = int(input(f"Enter the array element {i+1}:"))

    if(Search(n,arr,target)==-1):
        print("the element is not found")
    else:
        print("The element is found at:",Search(n,arr,target)+1)


